apps/teams-test-app/src/App.tsx

```diff
@@ -16,29 +16,61 @@ const urlParams = new URLSearchParams(window.location.search);
 const getOriginsParam = urlParams.has('origins') && urlParams.get('origins') ? urlParams.get('origins') : '';
 const validMessageOrigins: string[] | undefined = getOriginsParam ? getOriginsParam.split(',') : undefined;
 
-// This is added for custom initialization when app can be initialized based upon a trigger/click.
-if (!urlParams.has('customInit') || !urlParams.get('customInit')) {
-  if (isTestBackCompat()) {
-    initialize(undefined, validMessageOrigins);
+if (!urlParams.has('testInit') || !urlParams.get('testInit')) {
```

Comment:
>testInit [](http://example.com/codeflow?start=20&length=8)

Is it possible to name this parameter in a way that is more description of what it does? I.e. "testInit" doesn't exactly say what behavior is being triggered. Perhaps the parameter could be named something like "doNotCallInitializeFunction" ? I'm not actually sure what this parameter is doing... except telling it to ignore the customInit parameter? Could we just not send the customInit parameter in situations where you don't want the behavior it triggers?

Reply:
There is a possibility I will remove this name because `customInit` and `appInitializationTest` url parameters do the same thing as it does.